### PR TITLE
Models graph hotfix

### PR DESCRIPTION
**Pull request by maamalama**

### CODE REVIEW

**Review by chitalian — APPROVED** [review]

File: web/components/templates/dashboard/useDashboardPage.tsx [hunk 1]
```diff
@@ -107,7 +118,7 @@ export const useDashboardPage = ({
     filterMap[modelFilterIdx] = {
       label: "Model",
       operators: textWithSuggestions(
-        topModels
+        allModelsData
```
Comment (chitalian) [inline web/components/templates/dashboard/useDashboardPage.tsx:121]: if topModels is not being used, let's delete it

File: web/components/templates/dashboard/useDashboardPage.tsx [hunk 2]
```diff
@@ -99,6 +99,17 @@ export const useDashboardPage = ({
...
+  const { isLoading: isAllModelsLoading, models: allModels } = useModels(
+    timeFilter,
+    1000,
+    []
+  );
+
+  const allModelsData =
+    allModels?.data
+      ?.sort((a, b) => (a.total_requests > b.total_requests ? -1 : 1))
+      .slice(0, 5) ?? [];
```
Comment (chitalian) [inline web/components/templates/dashboard/useDashboardPage.tsx:111]: I dont think we need this at all?

File: web/components/templates/dashboard/useDashboardPage.tsx [hunk 3]
```diff
@@ -99,6 +99,17 @@ export const useDashboardPage = ({
...
+
```
Comment (chitalian) [inline web/components/templates/dashboard/useDashboardPage.tsx:112]:
```suggestion
```

File: web/components/templates/dashboard/useDashboardPage.tsx (hunk 1 above)
Comment (chitalian) [inline web/components/templates/dashboard/useDashboardPage.tsx:121]:
```suggestion
        allModels.data
```

File: web/components/templates/dashboard/useDashboardPage.tsx (hunk 2 above)
Comment (maamalama) [reply web/components/templates/dashboard/useDashboardPage.tsx:111]: I think so, just displaying in order

File: web/components/templates/dashboard/useDashboardPage.tsx (hunk 1 above)
Comment (maamalama) [reply web/components/templates/dashboard/useDashboardPage.tsx:121]: we still need topModels for graph, and need allModels for filters

**Pull request merged**